src/containers/DAO/DAOContainer.tsx

```diff
@@ -52,7 +53,9 @@ const DAOContainer: React.FC<StakingContainerProps> = ({
 
 	return (
 		<main className={classNames(styles.Container, className)} style={style}>
-			{zna === '' ? <DAOList /> : <DAOPage />}
+			{zna === DAO_CREATE && <DAOCreate />}
+			{zna === '' && <DAOList />}
+			{zna !== DAO_CREATE && zna !== '' && <DAOPage />}
```

Comment:
It's hard to read the conditional rendering. Can you update with more readable/understandable conditions? 💪